src/server.ts

```diff
@@ -6,15 +7,11 @@ import ModeRegistry from './helpers/modeRegistry';
 import redisClient from './helpers/redisClient';
 import logger from './logger';
 
-import ManualMode from './modes/manual.mode/manual';
-
 // Initialize the redis client
 redisClient.initialize();
 
-const manualMode = new ManualMode();
-
 // Register the modes
-ModeRegistry.registerModes([manualMode]);
+ModeRegistry.modesRegisterFromDir(path.join(__dirname, 'modes'));
```

Comment:
Consider moving the directory path to a configuration file or environment variable. This can make the code more flexible and easier to manage. [medium]